src/components/datum.js

```diff
@@ -14,7 +14,16 @@ export default class Datum extends Component {
             var DATA_DECIMALS = 2;
         }
 
-        let str = parseFloat(this.props.data.data_value).toFixed(DATA_DECIMALS);
+        let str = this.props.data.data_value;
+        let float = parseFloat(str).toFixed(DATA_DECIMALS);
+
+        if(float != 'NaN') {
```

Comment:
Can we do:

```js
 let str = this.props.data.data_value;
 let float = parseFloat(str);

if (isNaN(float)) {
    // datum is not a number, leave formatted as string
} else {
    // datum is floating point, make fixed point
    str = float.toFixed(DATA_DECIMALS);
}
```

This uses the JavaScript `isNaN()` API to determine if the input string was numeric, rather than going through two levels of data type conversion and comparing as a string.

We could also check for whether "." appears in the input string, and if not, leave it as-is. This would preserve integer values.